tools/tf2openapi/cmd/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"github.com/golang/protobuf/proto"
+	"github.com/kubeflow/kfserving/tools/tf2openapi/generator"
+	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var (
+	model = flag.String("model", "", "Absolute path of SavedModel file")
+	out   = flag.String("out", "std", "Either 'std' to display on standard out or 'file' to save output")
```

Comment:
If you were to have an enum of options, you could make this as an enum (if possible).

Instead, let's just default to std::out and output to file if output-file provided.

Reply:
The flag pkg doesn't support validating an enum of values for a flag so going with the second option.